### DISCUSSION TITLE
How to reference image in a HTML tiddler?

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#5814] [Q&A] [answered]
I know we can `[img[xxx.png]]` in tid file, or `![](xxx.png)` in a MD file, but how to achieve this in an HTML file, to transclude local image in the wiki?

I'm trying to add an extension that allow https://github.com/webclipper/web-clipper to clip website into nodejs tiddlywiki, and store images locally.

This will not work:

```
<img src="xxx.png" />
```

![截屏2021-06-21 下午8 27 42](https://user-images.githubusercontent.com/3746270/122761739-24a83a00-d2cf-11eb-92a6-e962fc48b32f.png)

So is there a way to achieve this in NodeJS wiki and HTML wiki?

**Comment by Jermolene** [comment, marked as answer]
Hi @linonetwo 

> I know we can `[img[xxx.png]]` in tid file, or `![](xxx.png)` in a MD file, but how to achieve this in an HTML file, to transclude local image in the wiki?

I don't think that's possible at the moment.
